components/agenda-jeune/OngletAgendaBeneficiaire.tsx

```diff
@@ -0,0 +1,104 @@
+import { DateTime } from 'luxon'
+import React, { useEffect, useState } from 'react'
+
+import AgendaActionRow from './AgendaActionRow'
+
+import { AgendaRdvRow } from 'components/agenda-jeune/AgendaRdvRow'
+import { IntegrationPoleEmploi } from 'components/jeune/IntegrationPoleEmploi'
+import { SpinningLoader } from 'components/ui/SpinningLoader'
+import { Action } from 'interfaces/action'
+import { RdvListItem } from 'interfaces/rdv'
+import { fusionneEtTriActionsEtRendezVous } from 'presentation/AgendaJeunePresentationHelper'
+import { AgendaMetadata, AgendaService } from 'services/agenda.service'
+import { toShortDate } from 'utils/date'
+import { useDependance } from 'utils/injectionDependances'
+
+interface OngletAgendaBeneficiaireProps {
+  idJeune: string
+  isPoleEmploi: boolean
+  updateNombreEvenement: (nombre: number) => void
+}
+
+export function OngletAgendaBeneficiaire({
+  idJeune,
+  isPoleEmploi,
+  updateNombreEvenement,
+}: OngletAgendaBeneficiaireProps) {
+  const agendaService = useDependance<AgendaService>('agendaService')
+  const [actionsEtRendezVous, setActionsEtRendezVous] = useState<
+    Array<Action | RdvListItem>
+  >([])
+  const [metadata, setMetadata] = useState<AgendaMetadata>()
+
+  function instanceOfAction(object: any): object is Action {
+    return 'creator' in object && 'dateEcheance' in object
+  }
+
+  function instanceOfRdvListItem(object: any): object is RdvListItem {
+    return 'beneficiaires' in object && 'modality' in object
+  }
+
+  function actionEtRendezVousRows(): JSX.Element[] {
+    return actionsEtRendezVous.map((item) => {
+      if (instanceOfAction(item)) {
+        return <AgendaActionRow key={item.id} action={item} jeuneId={idJeune} />
+      }
+      if (instanceOfRdvListItem(item)) {
+        return <AgendaRdvRow key={item.id} rdv={item} />
+      }
+      return <></>
+    })
+  }
+
+  useEffect(() => {
+    if (!isPoleEmploi) {
+      agendaService
+        .recuperationAgendaMilo(idJeune, DateTime.now())
+        .then((response) => {
+          updateNombreEvenement(
+            response.actions.length + response.rendezVous.length
+          )
+          setActionsEtRendezVous(
+            fusionneEtTriActionsEtRendezVous(
+              response.actions,
+              response.rendezVous
+            )
+          )
+          setMetadata(response.metadata)
+        })
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [])
+
+  return (
+    <>
+      {isPoleEmploi && (
+        <IntegrationPoleEmploi label='convocations et démarches' />
+      )}
+
+      {!isPoleEmploi && !metadata && <SpinningLoader />}
+
+      {!isPoleEmploi && metadata && (
+        <div>
+          <p className='text-m-bold text-primary mb-6'>
+            Période du {toShortDate(metadata.dateDeDebut)} au{' '}
```

Comment:
```suggestion
            Semaine du {toShortDate(metadata.dateDeDebut)} au{' '}
```
a priori on est pas sur le même fonctionnement que l'agenda conseiller, où on est en période, mais ici en semaine. à vérifier

Reply:
temporaire , va changer dans next issue